source/fundamentals/builders.txt

```diff
@@ -272,6 +272,45 @@ single field path``, in this case using ``Category``:
          // Using string-based field names
          var keys = builder.Wildcard("Category");
 
+Build an Aggregation Pipeline
+-----------------------------
+
+The ``PipelineDefinitionBuilder`` class provides a type-safe interface for
+defining an **aggregation pipeline**. An aggregation pipeline is a series of
+stages that are used to transform a document. Suppose you want to create a
+pipline that performs the following operations:
+
+- Matches all documents with "spring" in the ``Season`` field
+- Sorts the results by the ``Category`` field
+- Groups the documents by category and shows the average price and total
+  available for all documents in that category
+
+Use ``PipelineDefinitionBuilder`` classes to build the pipeline:
+
+.. code-block:: csharp
+
+   var sortBuilder = Builders<Flower>.Sort.Ascending(f => f.Category);
```

Comment:
I would want to see a description of AppendStage method and his usage. Something like: If you don't find appropriate typed safe method in aggregation builder, you can still define your stage with a raw MQL stage. Additionally we can add a note about raw stage handling similar  to the last node on this page: https://mongodb.github.io/mongo-csharp-driver/2.18/frequently_encountered_issues/